pynestml/frontend/pynestml_frontend.py

```diff
@@ -117,12 +119,32 @@ def install_nest(target_path: str, nest_path: str) -> None:
     nest_path : str
         Path to the NEST installation, which should point to the main directory where NEST is installed. This folder contains the ``bin``, ``lib(64)``, ``include``, and ``share`` folders of the NEST install. The ``bin`` folder should contain the ``nest-config`` script, which is accessed by NESTML to perform the installation. This path is the same as that passed through the ``-Dwith-nest`` argument of the CMake command before building the generated NEST module. The suffix ``bin/nest-config`` will be automatically appended to ``nest_path``.
 
+    install_path: str
+        Path to the install directory, where the generated module library will be created.
+
     Raises
     ------
     GeneratedCodeBuildException
         If any kind of failure occurs during cmake configuration, build, or install.
     '''
-    nest_installer(target_path, nest_path)
+    if install_path is not None:
+        nest_installer(target_path, nest_path, install_path)
+        # add the install_path to the python library
+        system = platform.system()
+        lib_key = ""
+
+        if system == "Linux":
+            lib_key = "LD_LIBRARY_PATH"
+        else:
+            lib_key = "DYLD_LIBRARY_PATH"
+
+        lib_path = os.path.join(install_path, "lib", "nest")
```

Comment:
I think the expected behavior would be to do as the docstring says: use the path given by the user without any change.
Now I can somewhat understand that we might want to append ``/nest`` to it to prevent collisions but ``/lib``? I don't think anyone would expect that...

The user should be expected to pass ``./local/lib`` and we use it directly.
It might be OK to add ``/nest`` to it in addition but in that case this should really be stated very clearly in the docstring.